src/Bicep.Core/TypeSystem/TypeAssignmentVisitor.cs

```diff
@@ -633,12 +669,16 @@ public override void VisitVariableAccessSyntax(VariableAccessSyntax syntax)
 
                     case VariableSymbol variable:
                         return new DeferredTypeReference(() => VisitDeclaredSymbol(syntax, variable));
+                    
+                    case NamespaceSymbol _:
+                        // consider using a constant
```

Comment:
If it's only used in one place, we don't need a constant.

Reply:
Removed comment.